styles/kotlin/catppuccin.user.css

```diff
@@ -0,0 +1,697 @@
+      background-image: url("data:image/svg+xml,@{svg}");
+    }
+
+    .kt-header__product-version {
+      color: @text;
+    }
+
+    // CodeMirror and output styles
+    .theme-kotlin-docs-wrapper {
+      .cm-s-kotlin-docs.CodeMirror {
+        background-color: @base;
+      }
+
+      .js-code-output-executor,
+      .output-wrapper {
+        border-radius: 0 0 8px 8px;
+        background: @base !important;
+      }
+    }
+    .CodeMirror-gutters {
+      background-color: @base;
+    }
+    .standard-output {
+      color: @text;
+    }
+
+    // TOC styles
+    .toc-item {
+      &--highlighted {
+        background-color: @base;
+      }
+
+      &--selected {
+        background-color: @accent-color;
+        color: @base;
+      }
+
+      &:hover {
+        background-color: @mantle
+      }
+    }
+
+
+    /* Start styling your website here: */
+
+    /* If you need to specify an overwrite specific to a flavor
+    *  (e.g. only when the flavor is set to latte), you can use a `when` guard clause. */
+
+    & when (@lookup = latte) {
+      img[class*="hero_imageDesktop"] {
+        filter: none;
+        mix-blend-mode: darken;
+      }
+    }
+
+  }
+
+}
+
+
+/* prettier-ignore */
```

Comment:
As the comment says, this should not be formatted! Please reset to the condensed block used in the template.